

In file included from net/sunrpc/sched.c:21:
include/linux/suspend.h: In function `freeze_processes':
include/linux/suspend.h:71: warning: no return statement in function returning non-void

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-power4-akpm/include/linux/suspend.h |    2 --
 1 files changed, 2 deletions(-)

diff -puN include/linux/suspend.h~next-step-of-smp-support-fix-device-suspending-warning-fix-2 include/linux/suspend.h
--- 25-power4/include/linux/suspend.h~next-step-of-smp-support-fix-device-suspending-warning-fix-2	2004-06-28 22:20:09.256061888 -0700
+++ 25-power4-akpm/include/linux/suspend.h	2004-06-28 22:24:46.722880520 -0700
@@ -68,8 +68,6 @@ extern void pm_restore_console(void);
 
 #else
 static inline void refrigerator(unsigned long flag) {}
-static inline int freeze_processes(void) { BUG(); }
-static inline void thaw_processes(void) {}
 #endif	/* CONFIG_PM */
 
 #ifdef CONFIG_SMP
_
